erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -203,24 +194,71 @@ const BaseStack = (specInput) => {
     return parameters;
   };
 
+  const configureParameterForLayer = (parameters, config, layerId) => {
+    if (parameters.encodings[layerId] === undefined ||
+        config[layerId] === undefined) {
+      return parameters;
+    }
+    const newParameters = parameters;
+    newParameters.encodings[layerId].maxBitrate = config[layerId].maxBitrate;
+    if (config[layerId].active !== undefined) {
+      newParameters.encodings[layerId].active = config[layerId].active;
+    }
+    return newParameters;
+  };
+
+  const setBitrateForVideoLayers = (sender, spatialLayerConfigs) => {
+    if (typeof sender.getParameters !== 'function' || typeof sender.setParameters !== 'function') {
+      log.warning('message: Cannot set simulcast layers bitrate, reason: get/setParameters not available');
+      return;
+    }
+    let parameters = sender.getParameters();
+    Object.keys(spatialLayerConfigs).forEach((layerId) => {
+      if (parameters.encodings[layerId] !== undefined) {
+        log.warning(`message: Configure parameters for layer, layer: ${layerId}, config: ${spatialLayerConfigs[layerId]}`);
+        parameters = configureParameterForLayer(parameters, spatialLayerConfigs, layerId);
+      }
+    });
+    sender.setParameters(parameters)
+      .then((result) => {
+        log.debug(`message: Success setting simulcast layer configs, result: ${result}`);
+      })
+      .catch((e) => {
+        log.warning(`message: Error setting simulcast layer configs, error: ${e}`);
+      });
+  };
+
+  that.setSimulcastLayersConfig = (streamInput) => {
+    log.debug(`message: Maybe set simulcast Layers config, simulcast: ${JSON.stringify(that.simulcast)}`);
+    if (streamInput.hasSimulcast() && streamInput.getSimulcastConfig().spatialLayerConfigs) {
+      streamInput.stream.transceivers.forEach((transceiver) => {
+        if (transceiver.sender.track.kind === 'video') {
```

Comment:
I know this is not from this PR, but I've seen issues with Unified Plan because transceiver.sender can be null sometimes, can you add a check here to avoid issues?